tools/opt/opt.cpp

```diff
@@ -68,46 +68,55 @@ NOTE: The optimizer is a work in progress.
                Remap result ids to a compact range starting from %%1 and without
                any gaps.
   --inline-entry-points-exhaustive
-               Exhaustively inline all function calls in entry point functions.
-               Currently does not inline calls to functions with early return
+               Exhaustively inline all function calls in entry point call tree
+               functions. Currently does not inline calls to functions with
+               early return in a loop.
+  --inline-entry-points-opaque
+               Inline all function calls in entry point call tree functions
+               where the callee has opaque type parameters or return value.
```

Comment:
I'd clarify about what opaque means. Suggest "A type is considered opaque if it is an image, sampler, sampled image, is a pointer to one, or is a struct containing one."